src/main.cpp

```diff
@@ -155,8 +155,13 @@ QCoreApplication* createApplication(int& argc, char* argv[])
 #endif
         // Fix for display scaling like 125% or 150% on Windows
 #if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+#if !(defined(NO_VS) && defined(_WIN32))
         QGuiApplication::setHighDpiScaleFactorRoundingPolicy(
             Qt::HighDpiScaleFactorRoundingPolicy::PassThrough);
+#else
```

Comment:
Shouldn't this be `#elif !(defined(NO_VS))` ?

Reply:
PassThrough works fine in linux (and in fact is needed in some cases). It's really only when we're compiling NO_VS for windows that we want it set otherwise. (The pull request title is a little out of date.)

Reply:
Oh sorry, I confused NO_VS with NO_GUI.